# Housing Market

### Introduction:

This time we will create our own dataset with fictional numbers to describe a house market. As we are going to create random data don't try to reason of the numbers.

### Step 1. Import the necessary libraries

In [37]:
import pandas as pd
import numpy as np

### Step 2. Create 3 differents Series, each of length 100, as follows: 
1. The first a random number from 1 to 4 
2. The second a random number from 1 to 3
3. The third a random number from 10,000 to 30,000

In [45]:
series1 = pd.Series(np.random.randint(1,4,100))

In [46]:
series2 = pd.Series(np.random.randint(1,3,100))

In [47]:
series3 = pd.Series(np.random.randint(10000,30000,100))

### Step 3. Let's create a DataFrame by joinning the Series by column

In [48]:
DataFrame = pd.DataFrame(pd.concat([series1,series2,series3],axis=1))
DataFrame

0  1      2
0   2  2  15477
1   2  2  20981
2   2  1  18362
3   2  1  11301
4   3  1  18416
.. .. ..    ...
95  1  1  21246
96  2  2  14079
97  2  1  22308
98  2  2  17410
99  2  2  18466

[100 rows x 3 columns]

### Step 4. Change the name of the columns to bedrs, bathrs, price_sqr_meter

In [51]:
DataFrame.columns = ['bedrs','bathrs','price_sqr_meter']
DataFrame

bedrs  bathrs  price_sqr_meter
0       2       2            15477
1       2       2            20981
2       2       1            18362
3       2       1            11301
4       3       1            18416
..    ...     ...              ...
95      1       1            21246
96      2       2            14079
97      2       1            22308
98      2       2            17410
99      2       2            18466

[100 rows x 3 columns]

### Step 5. Create a one column DataFrame with the values of the 3 Series and assign it to 'bigcolumn'

In [52]:
bigcolumn = pd.DataFrame(pd.concat([series1,series2,series3],ignore_index=True,axis=0))
bigcolumn

0
0        2
1        2
2        2
3        2
4        3
..     ...
295  21246
296  14079
297  22308
298  17410
299  18466

[300 rows x 1 columns]

### Step 6. Oops, it seems it is going only until index 99. Is it true?

### Step 7. Reindex the DataFrame so it goes from 0 to 299

In [53]:
bigcolumn.reset_index()

index      0
0        0      2
1        1      2
2        2      2
3        3      2
4        4      3
..     ...    ...
295    295  21246
296    296  14079
297    297  22308
298    298  17410
299    299  18466

[300 rows x 2 columns]